ngmix/prepsfmom.py

```diff
@@ -42,6 +42,14 @@ def __init__(self, fwhm, kernel, pad_factor=4, ap_rad=1.5):
         self.pad_factor = pad_factor
         self.kernel = kernel
         self.ap_rad = ap_rad
+        if self.kernel == "ksigma":
+            self.kind = "ksigma"
```

Comment:
I think this makes the `kind` set in the inheriting classes redundant

Reply:
Also not this makes it impossible to do inheritance, since the type won't be "registered' 

I'm not sure what the right thing to do is.

Reply:
Oh, you're saying that in the inherited classes it's a class variable, but this setting is an instance variable?

Reply:
> Also not this makes it impossible to do inheritance, since the type won't be "registered'

What python foo is this?

Reply:
Really all I meant is what Eli said.

Reply:
My point about inheritance was just that these two options have special status ksigma and pgauss, and if you tried to make your own the base class would not know about it.

Reply:
Ahhh right. In our case the baee class errors and you need to adjust the guts. The inheritance here is more about the interface then extensibility.

Reply:
Probably this but I'm not gonna go down this path. 

https://stackoverflow.com/questions/45400284/understanding-init-subclass

Reply:
meh